src/main/webapp/app/exercises/programming/manage/code-editor/code-editor-instructor-base-container.component.ts

```diff
@@ -139,9 +139,9 @@ export abstract class CodeEditorInstructorBaseContainerComponent implements OnIn
                         this.exercise.exerciseHints = exerciseHints;
                         this.loadingState = LOADING_STATE.CLEAR;
                     },
-                    (err) => {
+                    (err: Error) => {
                         this.loadingState = LOADING_STATE.FETCHING_FAILED;
-                        this.onError(err);
+                        this.onError(err.message);
```

Comment:
@julian-christl E.g. in this case the subscription error-block actually expected a `string` instead of a proper error. Therefore, throwing just a string above was fine.
As there were only a few places where strings instead of errors were used, I replaced them with proper errors to be consistent everywhere.
Sadly there is no type  safety between the throws and the catch blocks.